src/zulip.rs

```diff
@@ -71,8 +75,6 @@ struct Response {
     content: String,
 }
 
-pub const BOT_EMAIL: &str = "triage-rust-lang-bot@zulipchat.com";
```

Comment:
I wanted to also set this global but then it needs to be cloned multiple times, so unsure what's the best solution